docs/how-to-guides/backup-rook-ceph-volumes.md

```diff
@@ -0,0 +1,264 @@
+`velero` user.
+
+#### Deploy
+
+Execute the following command to deploy the `velero` component:
+
+```bash
+lokoctl component apply velero
+```
+
+Verify the pods in the `velero` namespace are in the `Running` state (this may take a few minutes):
+
+```console
+$ kubectl -n velero get pods
+NAME                     READY   STATUS    RESTARTS   AGE
+restic-c27rq             1/1     Running   0          2m
+velero-66d5d67b5-g54x7   1/1     Running   0          2m
+```
+
+### Step 2: Deploy sample workload
+
+If you already have an application you want to backup, then skip this step.
+
+Let us deloy a stateful application and save some demo data in it. Save the following YAML config in
+a file named `stateful.yaml`:
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: demo-ns
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app: demo-app
+  name: demo-app
+  namespace: demo-ns
+spec:
+  replicas: 1
+  serviceName: "demo-app"
+  selector:
+    matchLabels:
+      app: demo-app
+  template:
+    metadata:
+      labels:
+        app: demo-app
+    spec:
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 65534
+        runAsGroup: 65534
+      containers:
+      - image: busybox:1
+        name: app
+        command: ["/bin/sh"]
+        args:
+        - -c
+        - "echo 'sleeping' && sleep infinity"
+        volumeMounts:
+        - mountPath: "/data"
+          name: data
+  volumeClaimTemplates:
+  - metadata:
+      name: data
+    spec:
+      accessModes:
+      - ReadWriteOnce
+      resources:
+        requests:
+          storage: 100Mi
+```
+
+Execute the following command to deploy the application:
+
+```bash
+kubectl apply -f stateful.yaml
+```
+
+Verify the application is running fine:
+
+```console
+$ kubectl -n demo-ns get pods
+NAME         READY   STATUS    RESTARTS   AGE
+demo-app-0   1/1     Running   0          16s
+```
+
+Execute the following command to generate some dummy data:
+
+```console
+kubectl -n demo-ns exec -it demo-app-0 -- /bin/sh -c \
+    'dd if=/dev/zero of=/data/file.txt count=40 bs=1048576'
+```
+
+Verify that the data is generated:
+
+```console
+$ kubectl -n demo-ns exec -it demo-app-0 -- /bin/sh -c 'du -s /data'
+40960   /data
+```
+
+### Step 3: Annotate pods
+
+Annotate the pods with volumes attached to them with their volume names. Replace the pod name and
+the volume name as needed in the following command:
+
+```bash
+kubectl -n demo-ns annotate pod demo-app-0 backup.velero.io/backup-volumes=data
+```
+
+### Step 4: Backup entire namespace
+
+Execute the following command to start the backup of the concerned namespace. In our demo
+application's case it is `demo-ns`:
+
+```bash
+velero backup create backup-demo-app-ns --include-namespaces demo-ns --wait
+```
+
+Above operation may take some time, depending on the size of the data.
+
+### Step 5: Restore Volumes
```

Comment:
We could also mention restoration using different storage class: https://velero.io/docs/v1.5/restore-reference/#changing-pvpvc-storage-classes + https://github.com/vmware-tanzu/velero/issues/2971, but maybe this is too complex scenario, not sure.

Reply:
Yes I think it will be way to complex and not sure if it even fits in what we support.